### PR TITLE
Arrangement in the Author of my commits

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: l2j/game/data/scripts/custom/VotingReward/model/Droplist.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2004-2013 L2J Server
+ * Copyright (C) 2004-2015 L2J Server
```
Comment (lordrex34) [inline l2j/game/data/scripts/custom/VotingReward/model/Droplist.java]: 2017 please

File: l2j/game/data/scripts/custom/VotingReward/model/Droplist.java [hunk 2]
```diff
@@ -81,4 +81,4 @@ public boolean hasDrops()
 		}
 		return itemsToDrop != null ? itemsToDrop : Collections.<DroplistItem> emptyList();
 	}
-}
```
Comment (lordrex34) [inline l2j/game/data/scripts/custom/VotingReward/model/Droplist.java]: why?

File: l2j/game/data/scripts/custom/VotingReward/model/DroplistGroup.java [hunk 3]
```diff
@@ -49,4 +49,4 @@ public void addItem(DroplistItem item)
 		}
 		_items.add(item);
 	}
-}
+}
```
Comment (lordrex34) [inline l2j/game/data/scripts/custom/VotingReward/model/DroplistGroup.java]: why?

File: l2j/game/data/scripts/custom/VotingReward/model/DroplistItem.java [hunk 4]
```diff
@@ -61,4 +61,4 @@ public double getChance()
 	{
 		return _chance;
 	}
-}
+}
```
Comment (lordrex34) [inline l2j/game/data/scripts/custom/VotingReward/model/DroplistItem.java]: why?

File: l2j/game/data/scripts/custom/VotingReward/model/ItemHolder.java [hunk 5]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2004-2014 L2J DataPack
+ * Copyright (C) 2004-2015 L2J DataPack
```
Comment (lordrex34) [inline l2j/game/data/scripts/custom/VotingReward/model/ItemHolder.java]: 2017 please

File: l2j/game/data/scripts/custom/VotingReward/model/ScopeContainer.java [hunk 6]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2004-2014 L2J DataPack
+ * Copyright (C) 2004-2015 L2J DataPack
```
Comment (lordrex34) [inline l2j/game/data/scripts/custom/VotingReward/model/ScopeContainer.java]: 2017 please

File: l2j/game/data/scripts/custom/VotingReward/VotingReward.java [hunk 7]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2004-2014 L2J DataPack
+ * Copyright (C) 2004-2015 L2J DataPack
```
Comment (lordrex34) [inline l2j/game/data/scripts/custom/VotingReward/VotingReward.java]: 2017 please

File: l2j/game/data/scripts/custom/VotingReward/model/UserScope.java [hunk 8]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2004-2014 L2J DataPack
```
Comment (lordrex34) [inline l2j/game/data/scripts/custom/VotingReward/model/UserScope.java]: 2017 please

File: l2j/game/data/scripts/custom/VotingReward/util/DocumentParser.java [hunk 9]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2004-2014 L2J Server
```
Comment (lordrex34) [inline l2j/game/data/scripts/custom/VotingReward/util/DocumentParser.java]: 2017 please

**Review by lordrex34 — CHANGES_REQUESTED** [review]
In short: submitting mostly whitespace changes with minor compatibility changes.

You should update the years also.

**Comment by UnAfraid** [comment]
U didn't made the pull requests properly, you're including my own commits, when u do it right open a new one.

**Comment by ghost** [comment]
Okey :+1  as we talk by other way... we close this MR.

**Pull request closed without merging**